documentation/src/main/asciidoc/integrationguide/chapters/services/extras/extend/EventPublishingServiceContributor.java

```diff
@@ -3,10 +3,10 @@ public class EventPublishingServiceContributor
 
     @Override
     public void contribute(StandardServiceRegistryBuilder builder) {
-        builder.addinitiator( eventpublishingserviceinitiator.instance );
+        builder.addInitiator( EventPublishingServiceInitiator.instance );
 
         // if we wanted to allow other strategies (e.g. a jms
-        // queue publisher) we might also register short names
+        // queue publisher) we might also register short-names
```

Comment:
Hyphenating "short names" is just weird. You don't need a hyphen between an adjective and a noun.

A hyphen would be used if this were a compound adjective, but it's not. So if there are other occurrences of "short-name", they should be fixed by removing the hyphen.

Reply:
I changed other 'short-name's to 'short name's as you suggested, in addition to this one.